### DISCUSSION TITLE
Two Image Questions

### DISCUSSION THREAD

**Discussion by djotaku** [gohugo-ananke/ananke#767] [2 - Q&A / Support]
1. Are image sizes listed somewhere? If I want to have the max size of my images match the max size (and ratio) for the header and post images, how do I know what size to pick?
2. Is there a way, within Ananke, to set up images for Ananke's responsive design? If I wanted to have pre-rendered images for computer, tablet, and phone - is there a way to do it with the current Ananke design?

Thanks!

**Comment by davidsneighbour** [comment]
1) No, not yet.
2) No, not yet.

Responsive images are something I want to add quite soon. I have [my own plugin](https://github.com/davidsneighbour/hugo-modules/tree/main/modules/pictures) that takes care of these things on my other sites and think the best would probably be using that, as soon as the documentation is ready. For now I don't see how to find a max size, because the header images are all full size, which depends on the view port. So adding proper image tags with various sizes up to a configured max looks like the way to go.

**Comment by djotaku** [comment]
@davidsneighbour thanks for the work you are doing w/ the theme. I'll look for your plugin once you're ready with the documentation. 

That said, is there not a max width for the theme in posts? (On my full screen 2k monitor, it takes up maybe 1/4 of the width of the monitor) Is that listed somewhere in the theme? This way I could at least have my max image size match that size (independent of the responsive image thing).

Thanks!
